.github/workflows/push.yml

```diff
@@ -0,0 +1,35 @@
+name: Test suite workflow
+
+on:
+  # Opening a pull request to master branch will be a trigger.
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  Running-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ 3.5, 3.8 ]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Cache dependency # caching dependency will make our build faster.
+        uses: actions/cache@v2 # for more info checkout pip section documentation at https://github.com/actions/cache
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements/test.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Set up Python ${{ matrix.python-version }}
```

Comment:
Please, add a blank line before this line. @ivanvgh

Reply:
Done.